src/index.html

```diff
@@ -4,14 +4,182 @@
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="input-line">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="input-line">
+          Full date of birth:
+          <input
+            type="date"
+            name="FullDateOfbirth"
+          >
+        </label>
+
+        <label class="input-line">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="inputs-block">
+        <legend>Registration</legend>
+
+        <label class="input-line">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="input-line">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="1"
+            maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="inputs-block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="input-line">
+          <label>
+          Do you love cats?
+            <input
+              type="radio"
+              name="doYouLoveCats"
+              value="Yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="doYouLoveCats"
+              value="No"
+            >
+            NO
+          </label>
+
+        </div>
+        <label class="input-line">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+
+        <label class="input-line">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            step="1"
+          >
+        </label>
+
+        <label class="input-line">
+          What are your favorite brand of cars
+          <select multiple name="cars">
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="input-line">
+          How do you rate your work?
+          <input
+            type="range"
+            name="rate"
+            min="1"
+            max="5"
+            step="1"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="inputs-block">
+        <legend>Additional info:</legend>
+        <label class="input-line">
+        Comments:
+          <textarea
+            name="info"
+            autocomplete="off"
+         ></textarea>
```

Comment:
same here
```suggestion
          Comments:
          <textarea
            name="info"
            autocomplete="off"
           >
           </textarea>
```

Reply:
@anstsot 
ATTENTION!!! the closing tag should start on the level as opening tag (this is required to pass the html linter)
Example:

 <textarea
   class="textarea"
></textarea>